src/components/Home/HomeComponent.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import { Segment, Icon } from 'semantic-ui-react';
+import { withRouter} from 'react-router-dom';
```

Comment:
I apologize for being nitpicky but please provide a space after the `r` in `withRouter`. So it would be `{ withRouter }`.